pkg/diffs/diffs_test.go

```diff
@@ -20,7 +20,12 @@ import (
 	"github.com/openshift/ci-tools/pkg/jobconfig"
 )
 
-var ignoreUnexported = cmpopts.IgnoreUnexported(prowconfig.Presubmit{}, prowconfig.Brancher{}, prowconfig.RegexpChangeMatcher{})
+var ignoreUnexported = cmpopts.IgnoreUnexported(
+	prowconfig.Presubmit{},
+	prowconfig.Brancher{},
+	prowconfig.RegexpChangeMatcher{},
+	prowconfig.Periodic{},
```

Comment:
The whole thing!?!?!

Reply:
I understand this is how `IgnoreUnexported` is supposed to be used:

> IgnoreUnexported returns an Option that only ignores the immediate unexported fields of a struct

`Periodic` has an unexported `interval time.Duration` member, so I think this usage is correct.